Legacy/bonej/src/main/java/org/bonej/plugins/EllipsoidFactor.java

```diff
@@ -144,15 +140,14 @@ public void run(final String arg) {
 		gd.addNumericField("Gaussian_sigma", 2, 0, 4, "px");
 		gd.addCheckbox("Flinn_plot", false);
 
-		gd.addMessage("Ellipsoid Factor is beta software.\n" + "Please report your experiences to the user group:\n"
-				+ "http://bit.ly/bonej-group");
+		gd.addMessage("Ellipsoid Factor is beta software.\n" +
+			"Please report your experiences to the user group:\n" +
+			"http://bit.ly/bonej-group");
 		gd.addHelp("http://bonej.org/ef");
```

Comment:
should this not point to the imagej forum these days?

Reply:
Good catch